introduction.md

```diff
@@ -1 +1,14 @@
- Hi this is Sonya I really enjoy front-end development. My favorite courses are HTML & CSS + Angular.JS! 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?